recipes/qemu/meta.yaml

```diff
@@ -0,0 +1,99 @@
+{% set name = "qemu" %}
+{% set version = "5.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # if you download the source from GH archive it will be incomplete and you'll get
+  # ERROR: missing file $SRC_DIR/ui/keycodemapdb/README
+  #
+  # This is not a GIT checkout but module content appears to
+  # be missing. Do not use 'git archive' or GitHub download links
+  # to acquire QEMU source archives. Non-GIT builds are only
+  # supported with source archives linked from:
+  # 
+  #   https://www.qemu.org/download/#source
+  # 
+  # Developers working with GIT can use scripts/archive-source.sh
+  # if they need to create valid source archives.
+  #
+  #url: https://github.com/qemu/qemu/archive/refs/tags/v{{ version }}.tar.gz
+  url: https://download.qemu.org/qemu-{{ version }}.tar.xz
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - sysroot_linux-64 2.17  # [linux64]  see below regarding kernel-headers
+    - make
+    - pkg-config
+    - bison
+    - flex
+    - gdb
+    - python # needed to run some tests, not to link against
+  host:
+    - gtk3  # [linux] ?
+    - glib
+    - libfdt
```

Comment:
Seems like QEMU has a vendored libfdt because I see `-I$SRC_DIR/dtc/libfdt`.  Need to look into that.

Reply:
```suggestion
```
remove libfdt dep because it gets flagged as overlinking because qemu has a vendored version.